CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+## Unreleased
+
+Released YYYY-MM-DD.
+
+### Added
+
+* TODO (or remove section if none)
+
+### Changed
+
+* TODO (or remove section if none)
+
+### Deprecated
+
+* TODO (or remove section if none)
+
+### Removed
+
+* TODO (or remove section if none)
+
+### Fixed
+
+* TODO (or remove section if none)
+
+### Security
+
+* TODO (or remove section if none)
+
+--------------------------------------------------------------------------------
+
+## 0.7.0
+
+Released 2020-01-14.
+
+### Changed
+
+* New fuzz projects will use `libfuzzer-sys` from crates.io now, instead of a
```

Comment:
Probably mention all the cool stuff from libfuzzer and Arbitrary that got pulled in?